libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -16,19 +16,23 @@ $theme: (
 		fonts: (
 			hero-titles: (
 				family: '',
-				size: 0
+				size: 0,
+				letter-spacing: null
 			),
 			titles: (
 				family: '',
-				size: 0
+				size: 0,
+				letter-spacing: null
 			),
 			body: (
 				family: '',
```

Comment:
Have you tested if these changes work? You can execute tests with new properties.

Reply:
letter-spacing and font-size have been tested and both of them work properly after the last changes